dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.bnd

```diff
@@ -10,18 +10,16 @@
 #*******************************************************************************
 -include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-bindings-soap;3.4.1;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
-
-cxfVersion=3.4.1
```

Comment:
Can you re-add the variable? I think it's a good practice to save us from having to make multiple updates when we change versions.